circleguard/visualizer.py

```diff
@@ -353,31 +361,42 @@ def paint_frametime_graph(self):
 
     def draw_line(self, alpha, start, end):
         """
-        Draws a line using the given painter, pen, and alpha level from Point start to Point end.
+        Draws a line at the given alpha level from the start point to the end point.
 
         Arguments:
-            QPainter painter: The painter.
-            Integer alpha: The alpha level from 0.0-1.0 to set the line to.
+            Float alpha: The alpha level from 0.0-1.0 to set the line to.
                            https://doc.qt.io/qt-5/qcolor.html#alpha-blended-drawing
             List start: The X&Y position of the start of the line.
             List end: The X&Y position of the end of the line.
         """
-
         self.painter.setOpacity(alpha)
         self.painter.drawLine(self.scaled_point(start[0], start[1]), self.scaled_point(end[0], end[1]))
 
-    def draw_cross(self, alpha, point):
+    def draw_cross(self, alpha, point, grey_out, pen):
```

Comment:
`pen` argument is stupid. store the `pen` that was set before the switching to the grey_out pen. 
Also only set the pen back to its old one when grey_out is set to save perf